### DISCUSSION TITLE
不十分な自前データセット使用時のモデル選択について

### DISCUSSION THREAD

**Discussion by RRR-troisR** [espnet/espnet#3661] [Q&A] [answered]
500文弱ほどの自前のデータセットを用いてTTSモデルを学習させようと考えています。（コーパス等の文ではないため音素バランス等は不明）
今考えている選択肢がtacotoron2のファインチューニングもしくはVITSの使用です。一般的にはどちらのほうが精度が出るでしょうか？また、他の選択肢がありましたら教えていただきたいです。
加えてVITSのファインチューニングや転移学習についての話を聞いたことがないので何か知っていましたら教えて頂きたいです。

**Comment by kan-bayashi** [comment, marked as answer]
こんにちは、ESPnetをご利用いただきありがとうございます。
100文でVITS適応をいくつかの話者で検討しましたが、そこそこの品質で動きます。
VITSだとボコーダの方と別々にやらなくても良いので非常に楽です。

デモでいくつか試すことができます。
https://colab.research.google.com/github/espnet/notebook/blob/master/espnet2_tts_realtime_demo.ipynb
- 'kan-bayashi/jvs_jvs010_vits_prosody': jvs010での100文適応
- 'kan-bayashi/tsukuyomi_full_band_vits_prosody': つくよみちゃんコーパスでのフルバンド100文適応

サンプルをいくつか下の方で聞くことができます。
https://espnet.github.io/icassp2022-tts/

適応の手順はこの辺をどうぞ。
https://github.com/espnet/espnet/tree/master/egs2/jvs/tts1#how-to-run
